src/backend/distributed/sql/downgrades/citus--11.1-1--11.0-2.sql

```diff
@@ -46,4 +46,15 @@ CREATE FUNCTION pg_catalog.worker_repartition_cleanup(bigint)
  STRICT
 AS 'MODULE_PATHNAME', $function$worker_repartition_cleanup$function$;
 
+DROP TYPE IF EXISTS citus.split_mode;
+DROP TYPE IF EXISTS citus.split_copy_info;
```

Comment:
why wouldn't it exist?

Reply:
Was just trying to be defensive, will remove.